Backend/src/middlewares/passportOauth.ts

```diff
@@ -0,0 +1,37 @@
+import passport from 'passport';
+const path = require('path')
+require('dotenv').config({ path: path.join(process.cwd(), '.env') })
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+
+
+const initPassport = () => {
+    const GoogleTokenStrategyCallback = (accessToken: String, refreshToken: String, profile: any, done: Function) => {
+        console.log('\nACCESS TOKEN \n', accessToken);
+        console.log('\nREFRESH TOKEN\n', refreshToken);
+        console.log('\nPROFILE\n', profile);
+        // Calls for logic or simply logic for storing these tokens goes here
```

Comment:
So if we wanna, lets say, store the refreshtoken for the usecase where an android phone sends in an expired token, we can then get a new token via the refreshtoken. 

In that case, we add that logic here?

Reply:
yes!